### PR TITLE
WC2-173: entities clean up and cypress list tests

### PR DESCRIPTION
**Pull request by beygorghor**
Entities domain Cypress tests are skipped for now.
Also we have lot's of dead code in there

Related JIRA tickets : WC2-173

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new string have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- removed unused code
- moved beneficiaries code at the root of entities folder
- wrote entities test to test list page 

## How to test

Explain how to test your PR.
If a specific config is required explain it here (dataset, account, profile, ...)

## Print screen / video

Upload here print screens or videos showing the changes

## Notes

Cypress tests are successful [here](https://github.com/BLSQ/iaso/actions/runs/4403236501)

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/entities/components/Filters.tsx [hunk 1]
```diff
@@ -1,104 +1,179 @@
...
+        location: params.location,
+        dateFrom: params.dateFrom,
+        dateTo: params.dateTo,
+        submitterId: params.submitterId,
+        submitterTeamId: params.submitterTeamId,
+        entityTypeIds: params.entityTypeIds,
     });
+    const [filtersUpdated, setFiltersUpdated] = useState(false);
+    const [initialOrgUnitId, setInitialOrgUnitId] = useState(params?.location);
+    const [textSearchError, setTextSearchError] = useState<boolean>(false);
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/entities/components/Filters.tsx:62]: Separate ticket but: we could probably add this to `useFilterState`

File: hat/assets/js/cypress/integration/07 - entities/list.spec.js [hunk 2]
```diff
@@ -103,236 +122,141 @@ describe.skip('Entities', () => {
                     .should('equal', undefined);
             });
         });
-        // skipping until we figure how to articulate entities and beneficiaries
-        it.skip('action should deep link search', () => {
+
+        it('action should deep link search', () => {
```
Comment (quang-le) [inline hat/assets/js/cypress/integration/07 - entities/list.spec.js:208]: we can probably group these 3 tests and save the repetition of `beforeEach`

File: hat/assets/js/cypress/integration/07 - entities/list.spec.js [hunk 3]
```diff
@@ -103,236 +122,141 @@ describe.skip('Entities', () => {
...
+        cy.fillSingleSelect('#submitterId', 0);
+        cy.testInputValue('#submitterId', 'oneFistPunch (Bruce Lee) ');
+        cy.log('select a team');
+        cy.intercept('GET', '/api/profiles', {
+            fixture: 'profiles/list-not-paginated.json',
+        }).as('getProfiles');
+        cy.fillSingleSelect('#submitterTeamId', 0);
+        cy.wait('@getProfiles').then(() => {
+            cy.log('User should be empty again');
+            cy.testInputValue('#submitterId', '');
```
Comment (quang-le) [inline hat/assets/js/cypress/integration/07 - entities/list.spec.js]: 🤩

**Review by codeKameleon — CHANGES_REQUESTED** [review]
Thi test is failing on my side : "Filter button action should deep link search and call api with same params"



https://user-images.githubusercontent.com/25134301/224738784-8eab4223-0064-4faf-8188-5432e77c87de.mov

**Review by quang-le — APPROVED** [review]

**Pull request merged**